src/Libraries/Microsoft.Extensions.AI.Abstractions/Image/ImageGenerationOptions.cs

```diff
@@ -81,6 +81,11 @@ protected ImageGenerationOptions(ImageGenerationOptions? other)
     /// </summary>
     public ImageGenerationResponseFormat? ResponseFormat { get; set; }
 
+    /// <summary>
+    /// Gets or sets the number of intermediate streaming images to generate.
```

Comment:
This is only relevant for streaming requests, right? Might be worth a comment. Should partial or intermediate be in the name? Otherwise it seems like it could be confused with Count, like Count applies to non-streaming and this is the number to generate for streaming. Also, is this per image? Like, if I set Count to 4 and this to 5, will I get 5 or 20?